tm2/pkg/sdk/auth/test_common.go

```diff
@@ -31,13 +31,17 @@ func setupTestEnv() testEnv {
 	prmk := params.NewParamsKeeper(authCapKey)
 
 	acck := NewAccountKeeper(authCapKey, prmk.ForModule(ModuleName), std.ProtoBaseAccount)
+
 	bankk := NewDummyBankKeeper(acck, prmk.ForModule("dummybank"))
 	gk := NewGasPriceKeeper(authCapKey)
 
 	prmk.Register(ModuleName, acck)
 	prmk.Register("dummybank", bankk)
 
 	ctx := sdk.NewContext(sdk.RunTxModeDeliver, ms, &bft.Header{Height: 1, ChainID: "test-chain-id"}, log.NewNoopLogger())
+
+	acck.SetParams(ctx, DefaultParams()) // Setup default params
```

Comment:
I cannot begin to explain how much I hate the `setupTestEnv` function :)

I'm waiting for the moment where we use it for our entire node bootstrapping logic, since it's growing like a beast

Reply:
Resolve the convo, I'm just mentioning